packages/ui/src/components/sorts/index.ts

```diff
@@ -1,11 +1,13 @@
 import MultiSort from './multi-sort'
+import SimpleSort from './simple-sort'
 import { Sort as SortRoot } from './sort'
 import SortSelect from './sort-select'
 
 export const Sort = {
   Root: SortRoot,
   Select: SortSelect,
-  MultiSort: MultiSort
+  MultiSort: MultiSort,
+  SimpleSort: SimpleSort
```

Comment:
nit: If the names already convey the type (`SortSelect`, `SimpleSort`, etc.), there's no need to repeat `Sort` in each of them.

```
export const Sort = {
  Root: SortRoot,
  Select: SortSelect,
  Multi: MultiSort,
  Simple: SimpleSort
}

```